### PR TITLE
add prefer-namespace-keyword rule

### PR DESCRIPTION
**Pull request by disizali**
this PR is going to add :

+ `prefer-namespace-keyword rule`
+ `7 tests about it`

### CODE REVIEW

File: src/rules/prefer_namespace_keyword.rs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+pub struct PreferNamespaceKeyword;
+
+impl LintRule for PreferNamespaceKeyword {
+  fn new() -> Box<Self> {
+    Box::new(PreferNamespaceKeyword)
+  }
+
+  fn code(&self) -> &'static str {
+    "preferNamespaceKeyword"
```
Comment (bartlomieju) [inline src/rules/prefer_namespace_keyword.rs:17]: `prefer-namespace-keyword`

File: src/rules/prefer_namespace_keyword.rs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      .source_map
+      .span_to_snippet(mod_decl.span)
+      .expect("error in load snippet");
+
+    if let Some(capt) = KEYWORD.captures(&snippet) {
+      let keyword = capt.name("keyword").unwrap().as_str();
+      if keyword == "module" && !mod_decl.global {
+        self.context.add_diagnostic(
+          mod_decl.span,
+          "preferNamespaceKeyword",
```
Comment (bartlomieju) [inline src/rules/prefer_namespace_keyword.rs:61]: `prefer-namespace-keyword`

File: src/rules/prefer_namespace_keyword.rs (hunk 1 above)
Comment (disizali) [reply src/rules/prefer_namespace_keyword.rs]: done

File: src/rules/prefer_namespace_keyword.rs (hunk 2 above)
Comment (disizali) [reply src/rules/prefer_namespace_keyword.rs]: thanks for the notice , done 👍

**Pull request merged**